### DISCUSSION TITLE
Asus laptop bios flashing error

### DISCUSSION THREAD

**Discussion by Tykwha** [terminatorul/NvStrapsReBar#42] [Q&A] [answered]
After modify bios, i get  error "selected file is not a proper bios", can someone modify bios for me? Maybe i'm doing something wrong but i tried all ways
[FA506IVAS319(1).zip](https://github.com/terminatorul/NvStrapsReBar/files/14565118/FA506IVAS319.1.zip)

**Comment by UnidentifiedTag** [comment]
This is a manufacture block, you’ll need to flash the bios in freedos using a special tool 

Here is a quote from what another user did
> ```
> CPU: Intel Core i7 8086K
> Motherboard model: ASUS ROG STRIX Z370-E GAMING
> Motherboard chipset: Intel Z370
> Graphics card model: nVidia RTX 2060 SUPER
> GPU chipset: TU106
> GPU PCI VendorID:DeviceID (check GPU-Z): 10DE 1F06 
> GPU PCI subsystem IDs (check GPU-Z):  3842 3067
> VRAM size: 8GB
> New BAR size (GPU-Z): 8192MiB
> New BAR size (nvidia-smi):  8192MiB
> driver version: 551.76
> ```
> 
> This motherboard didn't support flashback so I had to use FreeDOS\AFUDOS 3.05.04 following the ASUS guide [here](https://winraid.level1techs.com/t/guide-how-to-flash-a-modded-ami-uefi-bios/30627/1), except replacing AFUWINx64.exe with AFUDOS.exe after booting into FreeDOS.
> 
> ![rebar success](https://private-user-images.githubusercontent.com/25919009/312008335-48a6bc72-2c18-4636-9a8d-0550ab420e8d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAyNDcxODEsIm5iZiI6MTcxMDI0Njg4MSwicGF0aCI6Ii8yNTkxOTAwOS8zMTIwMDgzMzUtNDhhNmJjNzItMmMxOC00NjM2LTlhOGQtMDU1MGFiNDIwZThkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzEyVDEyMzQ0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU4NzFlMmI1NjM0NzY5NDZmODcxNmExOGE2NWQ5NDJhNzU1ZjYxZTYxZTFiMWU1MzMwNmRiNTUzMzk5ZjI5MjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dqrYkI_0lJBQcuB9VfLNzqigIKYSdvKc9ax20TPycoc)

**Reply by UnidentifiedTag** [reply]
You do this at you’re own risk none of this is guaranteed to work

**Comment by xCuri0** [comment, marked as answer]
@Tykwha only way on laptops is hardware flasher most of the time unfortunately

but i even if you could flash through software i wouldn't flash without one, because there is no dual bios or flashback like on desktops

**Comment by Tykwha** [comment]
> @Tykwha only way on laptops is hardware flasher most of the time unfortunately
> 
> but i even if you could flash through software i wouldn't flash without one, because there is no dual bios or flashback like on desktop

Thanks for the answer, yes, I also eventually came to the same conclusion and have already flashed it using CH341A,
